CHANGELOG.md

```diff
@@ -85,3 +116,59 @@ Additions to existing modules
   quasiring                       : Quasiring c ℓ → Quasiring (a ⊔ c) (a ⊔ ℓ)
   commutativeRing                 : CommutativeRing c ℓ → CommutativeRing (a ⊔ c) (a ⊔ ℓ)
   ```
+
+* In `Function.Bundles.Bijection`:
+  ```agda
+  section          : B → A
+  inverseˡ         : Inverseˡ _≈₁_ _≈₂_ to section
+  strictlyInverseˡ : StrictlyInverseˡ _≈₂_ to section
+  inverseʳ         : Inverseʳ _≈₁_ _≈₂_ to section
+  strictlyInverseʳ : StrictlyInverseʳ _≈₁_ to section
+  ```
+
+* In `Function.Bundles.LeftInverse`:
+  ```agda
+  surjective       : Surjective _≈₁_ _≈₂_ to
+  surjection       : Surjection From To
+  ```
+
+* In `Function.Bundles.RightInverse`:
+  ```agda
+  isInjection      : IsInjection to
+  injective        : Injective _≈₁_ _≈₂_ to
+  injection        : Injection From To
+  ```
+
+* In `Function.Bundles.Surjection`:
+  ```agda
+  section          : B → A
+  inverseˡ         : Inverseˡ _≈₁_ _≈₂_ to section
+  strictlyInverseˡ : StrictlyInverseˡ _≈₂_ to section
+  ```
+
+* In `Function.Consequences`: the theory of the left inverse of a surjective function
+  ```agda
+  module Section (surj :  Surjective ≈₁ ≈₂ f)
+  ```
```

Comment:
Should this module be lifted out from `Consequences` to the top-level of `Function.*`? Or somewhere else?

Reply:
Given that the only property used in the `Section` module is `surjective⇒strictlySurjective`, it is tempting to inline that, against lifting out the whole module, and deprecating `Function.Consequences.*` entirely, as those properties are nowhere else used in `stdlib`? ... downstream PR... ?

Reply:
Hmmm... not 'nowhere':
* `Data.Product.Function.Dependent.Setoid`
* `Data.Nat.Binary.Properties`